src/assisted_test_infra/download_logs/__main__.py

```diff
@@ -89,6 +99,11 @@ def handle_arguments():
         type=str,
         default=os.path.join(os.getenv("HOME"), ".kube/config"),
     )
+    parser.add_argument(
```

Comment:
I think we should make it optional to avoid errors as we can't merge this PR and the other one at the same time

Reply:
any argument with -- preceding it is optional by default